VergeiOS/Utils/Platform.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  Platform.swift
+//  VergeiOS
+//
+//  Created by Swen van Zanten on 27-07-18.
+//  Copyright © 2018 Verge Currency. All rights reserved.
+//
+
+import Foundation
+
+struct Platform {
+    static let isSimulator: Bool = {
+        var isSim = false
+        #if arch(i386) || arch(x86_64)
```

Comment:
https://stackoverflow.com/a/30284266
For me, this seems to be the way cleaner solution to be honest 
```swift
import Foundation

struct Platform {

    static var isSimulator: Bool {
        return TARGET_OS_SIMULATOR != 0
    }

}
```

Reply:
A: You are right
B: Changing it now!